src/py42/services/cases.py

```diff
@@ -0,0 +1,194 @@
+from collections import OrderedDict
+
+from py42.services import BaseService
+from py42.services.util import get_all_pages
+from py42.settings import items_per_page
+
+
+class CasesService(BaseService):
+    """`Rest documentation https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases`__ ."""
+
+    _uri_prefix = "/api/v1/case"
+
+    def __init__(self, connection):
+        super(CasesService, self).__init__(connection)
+
+    def create(self, name, subject, assignee, description, findings):
+        """Creates a new case.
+
+        Args:
+            name (str): Name of the case.
+            subject (str): A subject of the case.
+            assignee (str): User UID of the assignee.
+            description (str): Description of the case
+            findings (str): Observations of the case.
+
+        Returns
+            :class:`py42.response.Py42Response`
+        """
+        data = {
+            "assignee": assignee,
+            "description": description,
+            "findings": findings,
+            "name": name,
+            "subject": subject,
+        }
+        return self._connection.post(self._uri_prefix, data)
+
+    def get_page(
+        self,
+        name=None,
+        status=None,
+        created_at=None,
+        updated_at=None,
+        subject=None,
+        assignee=None,
+        page_number=1,
+        page_size=items_per_page,
+        sort_direction="asc",
+        sort_key="number",
+        **kwargs
+    ):
+        """Gets an individual page of cases.
+
+        Args:
+            name (str, optional): Filter results by case name, matches partial names. Defaults to None.
+            status (str, optional): Filter results by case status, "OPEN" or "CLOSED". Defaults to None.
```

Comment:
We should consider including an enum `CaseStatus`

Reply:
Enums were not available until python3.4, I have added the changes but py2.7 is complaining `enum not found`. 
One of the solution is to add dependency of `enum34` module for py2.7. Do we want to add it as dependency?